indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -352,6 +353,18 @@ def reviewing_state(self):
         else:
             return AbstractReviewingState.mixed
 
+    @property
+    def data_by_field(self):
+        return {value.contribution_field_id: value for value in self.field_values}
+
+    @property
+    def submitted_for_tracks_titles(self):
+        return [x.title for x in self.submitted_for_tracks]
+
+    @property
+    def proposed_for_tracks_titles(self):
+        return [x.title for x in self.proposed_for_tracks]
```

Comment:
The new `proposed_for_tracks` table is something different (reviews that propose "other tracks", i.e. a many-to-many relationship between AbstractReview and Track)